ui/StatusQ/src/StatusQ/Core/Theme/StatusLightTheme.qml

```diff
@@ -18,6 +18,10 @@ ThemePalette {
     dangerColor2: getColor('red', 0.2)
     dangerColor3: getColor('red', 0.1)
 
+    warningColor1: getColor('orange')
```

Comment:
Did you forget to push the correct color?  :) (`#F6793C`)

Reply:
just pushed